packages/serverpod_client/lib/src/serverpod_client_browser.dart

```diff
@@ -33,34 +29,20 @@ abstract class ServerpodClient extends ServerpodClientShared {
     _httpClient = http.Client();
   }
 
-  Future<void> _initialize() async {
-    _initialized = true;
-  }
-
   @override
-  Future<T> callServerEndpoint<T>(
-      String endpoint, String method, Map<String, dynamic> args) async {
-    if (!_initialized) await _initialize();
```

Comment:
This has no effect besides potentially relinquishing the thread to the task runner before continuing. I assume that is not necessary, in which case this is dead code that should perhaps be removed.